### PR TITLE
Update Category Forms for Accessibility and API & Webhooks

### PR DESCRIPTION
**Pull request by ghostinhershell**
Removed <title> def and updated the topic order for the API & Webhooks and Accessibility category forms.

### CODE REVIEW

**Comment by Alokrar** [comment]
See

File: .github/DISCUSSION_TEMPLATE/accessibility.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-title: "<title>"
+title: " "
```
Comment (otter-computer) [inline .github/DISCUSSION_TEMPLATE/accessibility.yml:1]:
```suggestion
```

File: .github/DISCUSSION_TEMPLATE/api-and-webhooks.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
-title: "<title>"
+title: " "
```
Comment (otter-computer) [inline .github/DISCUSSION_TEMPLATE/api-and-webhooks.yml:1]:
```suggestion
```

**Review by otter-computer — CHANGES_REQUESTED** [review]
You'll need to remove the whole line here, not just change it to a blank space!

File: .github/DISCUSSION_TEMPLATE/api-and-webhooks.yml (hunk 2 above)
Comment (roymoses07) [reply .github/DISCUSSION_TEMPLATE/api-and-webhooks.yml:1]: Moses Ojero

**Comment by Alokrar** [comment]
Thanks

**Review by perezz281 — COMMENTED** [review]
#49380

**Comment by comet19950902** [comment]
As I can see, good.
Thank you.

**Comment by SuphathatUdomkan** [comment]
ghostinhershell-category-form-update

**Comment by dirrthick** [comment]
https://github.com/blockchain/My-Wallet-V3-iOS/compare/v202212.1.2-rc.13742...v202301.2.0-rc.13885

File: .github/DISCUSSION_TEMPLATE/accessibility.yml (hunk 1 above)
Comment (P-Tanawat) [reply .github/DISCUSSION_TEMPLATE/accessibility.yml:1]: [Auto Close PRs](https://github.com/community/community/actions/workflows/auto-close-pr.yml)
.diff

File: .github/DISCUSSION_TEMPLATE/api-and-webhooks.yml (hunk 2 above)
Comment (P-Tanawat) [reply .github/DISCUSSION_TEMPLATE/api-and-webhooks.yml:1]:
```[![Auto Close PRs](https://github.com/community/community/actions/workflows/auto-close-pr.yml/badge.svg)](https://github.com/community/community/actions/workflows/auto-close-pr.yml)
title: "Auto Close PRs"
```

**Review by P-Tanawat — COMMENTED** [review]
[Auto Close PRs](https://github.com/community/community/actions/workflows/auto-close-pr.yml)
[react-new](https://codesandbox.io/embed/react-new?fontsize=14&hidenavigation=1&theme=light) 
.diff

**Comment by TOUFIKTOUHAMI** [comment]
y

**Comment by TOUFIKTOUHAMI** [comment]
@
Duplicate of #sq

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

**Comment by TOUFIKTOUHAMI** [comment]
sasa

File: .github/DISCUSSION_TEMPLATE/api-and-webhooks.yml [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 labels: [API and Webhooks]
 body:
 - type: dropdown
   attributes:
     label: Select Topic Area
     description: What would you like to discuss?
     options:
-      - "Product Feedback"
       - "Question"
+      - "Product Feedback"
```
Comment (Mroczek1992) [inline .github/DISCUSSION_TEMPLATE/api-and-webhooks.yml:10]: Chciałbym bym porozmawiać

**Review by Nward8118 — COMMENTED** [review]
Duplicate of #

> t`
Nicholasward81@gmail.com `

**Pull request merged**